src/test/java/edu/cornell/kfs/module/purap/fixture/VendorCreditMemoDocumentFixture.java

```diff
@@ -1,43 +1,62 @@
 package edu.cornell.kfs.module.purap.fixture;
 
-import java.sql.Date;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
 
+import org.easymock.EasyMock;
+import org.kuali.kfs.krad.service.DocumentService;
 import org.kuali.kfs.module.purap.document.VendorCreditMemoDocument;
+import org.kuali.kfs.sys.businessobject.FinancialSystemDocumentHeader;
 import org.kuali.kfs.sys.context.SpringContext;
 import org.kuali.kfs.sys.document.AccountingDocumentTestUtils;
 import org.kuali.rice.core.api.datetime.DateTimeService;
 import org.kuali.rice.core.api.util.type.KualiDecimal;
 import org.kuali.rice.kew.api.exception.WorkflowException;
-import org.kuali.kfs.krad.service.DocumentService;
+
+import edu.cornell.kfs.module.cam.CuCamsTestConstants;
+import edu.cornell.kfs.module.purap.document.CuVendorCreditMemoDocument;
 
 public enum VendorCreditMemoDocumentFixture {
 
-	VENDOR_CREDIT_MEMO("Description", 0, 4291, "12345", "12-5-2014",
-			new KualiDecimal(100));
+	VENDOR_CREDIT_MEMO("Description", 0, 4291, "12345", "12-05-2014",
```

Comment:
This fixture constant is used by CuPurchasingAccountsPayableModuleServiceImplTest, which is not in the scope for this user story. I only edited the date string for consistency with the new enum constants below it, and I do not believe the associated tests are actively using the date value on this particular constant. If you think this constant should be left as-is, let me know and I can undo the change.